sql/core/src/test/resources/sql-functions/sql-expression-schema.md

```diff
@@ -1,4 +1,8 @@
 <!-- Automatically generated by ExpressionsSchemaSuite -->
+## Summary
```

Comment:
This is no longer in the golden file https://github.com/apache/spark/commit/986b0f769b8ffa8a033d0f182217e83faa38fb4a